### ISSUE TITLE
Pasting from Clipboard produces unfinished Paste.

### ISSUE THREAD

**Issue by Hexical1** [IntellectualSites/FastAsyncWorldEdit-1.13#93]
# Bug report for FastAsyncWorldEdit 1.13.x

**Debug paste link**: https://incendo.org/paste/view/78814bfd6f3343cab1c3c7a511fb48a2

**Description of the problem:** Pasting something from clipboard leaves giant spots unpasted.

Before Copy / Paste: https://imgur.com/gPvPQT8
After: https://imgur.com/86vVVMK

**How to replicate**: Copy a build, paste it somewhere else.

**Checklist**:

- [x] I included a `/fawe debugpaste` link
- [x] I made sure there aren't duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of [FAWE for 1.13.x](https://ci.athion.net/job/FAWE-1.13/)
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by Tsoccerguy3** [comment]
you can not have Worldedit and FAWE together , remove worldedit .

**Comment by Hexical1** [comment]
> you can not have Worldedit and FAWE together , remove worldedit .

WorldEdit isn't installed, the debugpaste may have picked up a world-edit file because i have it set to .JARold, but world-edit isn't enabled.

**Comment by Tsoccerguy3** [comment]
not every build is fit to use on a server , FAWE is under going a massive update . remove the worldedit just in case its the problem . Always test yours plugins in a local test server before deploying . Linux may ignore extensions

**Comment by Hexical1** [comment]
> not every build is fit to use on a server , FAWE is under going a massive update

Yes.... which is why im using a dev build, so I can report bugs... like this....

> remove the worldedit just in case its the problem . Always test yours plugins in a local test server before deploying . Linux may ignore extensions

World-Edit is gone, the problem still persists, Linux doesn't just magically decide to read .JARold files, and never has. 

This is a local test server, which is again - why its installed in the first place.

**Comment by Tsoccerguy3** [comment]
I just tested the the current compile and //cut , //copy and //paste was working although there are other issues .

**Comment by PhanaticD** [comment]
from what i tested it stopped pasting when it hit a chest, so maybe its chest or possible tile entities in general

**Comment by Hexical1** [comment]
> from what i tested it stopped pasting when it hit a chest, so maybe its chest or possible tile entities in general

Hmm not the case for me, your problem might be more related to this then 
https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/issues/90

I don't think my build has any chests, hoppers, or entities - its just blocks as far as I can tell.
